editor/components/button/index.js

```diff
@@ -16,6 +16,7 @@ function Button( { isPrimary, isLarge, isToggled, className, ...additionalProps
 		<button
 			type="button"
 			{ ...additionalProps }
```

Comment:
What I mean is that the `buttonRef` prop is passed here to the `button` element. And it's not a valid prop for `button`. So to avoid this, we should deconstruct `buttonRef` in the function props to omit it from `additionalProps`.

Something like:

`function Button( { isPrimary, isLarge, isToggled, className, buttonRef, ...additionalProps } ) {`